dicetable.py

```diff
@@ -0,0 +1,327 @@
+
+
+class dicetable(object):
```

Comment:
Class names should start with a capital. It's you choice this could be named "Dicetable" or "DiceTable" perhaps the later, but I could be convinced either way: https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming